lua/nvim-web-devicons/icons-default.lua

```diff
@@ -1958,6 +2000,18 @@ local icons_by_file_extension = {
     cterm_color = "255",
     name = "WebOpenFontFormat",
   },
+  ["x"] = {
+    icon = "",
+    color = "#599eff",
+    cterm_color = "111",
+    name = "Tweak",
+  },
+  ["xm"] = {
+    icon = "",
+    color = "#519aba",
+    cterm_color = "74",
+    name = "Tweak",
```

Comment:
Are these ones also iphone jailbreak conventions, or objective C?

`.x` is for lexx, although that editor is pretty much dead, so we can discount that.

Name decision:
1. Objective C
2. IOSJailbreak

Reply:
Hmmm...
Although I'm not a tweak dev I can tell that most .x files are Tweak.x which why I named it Tweak.
I'll ask some tweak devs for advice.

Reply:
Objective-C is probably the second best option but definently not IOSJailbreak
because first it probably should be iOS Jailbreak instead of IOSJailbreak and second tweaks aren't equal to jailbreak because jailbreaks try to catch iOS' vulnerbilities and bypass through them, on the other hand tweaks are made for iOSes that are already jailbroken
ill also ask other developers about `mm` `m` `xm` as well

Reply:
Ok I've ask my friend and it seems that
.xm .x are Logos
.m is obj-c
.mm is obj-c++

I think I'll keep them as these languages and not "Tweaks"

However both obj-c and obj-c++ doesn't have the icon rn so I'll have the original cpp and c icon as a replacement

Reply:
> Ok I've ask my friend and it seems that .xm .x are Logos .m is obj-c .mm is obj-c++
> 
> I think I'll keep them as these languages and not "Tweaks"
> 
> However both obj-c and obj-c++ doesn't have the icon rn so I'll have the original cpp and c icon as a replacement

Sounds great! Let's get this in.

Reply:
ok ill fix that now

Reply:
done
the dollar sign refers to the producer of logos theos
checkout https://theos.dev their logo is a dollar sign

Reply:
It still say's `Tweak`.